packages/create-astro/src/actions/template.ts

```diff
@@ -81,11 +86,18 @@ export default async function copyTemplate(tmpl: string, ctx: Context) {
 		} catch (err: any) {
 			fs.rmdirSync(ctx.cwd);
 			if (err.message.includes('404')) {
-				await error('Error', `Template ${color.reset(tmpl)} ${color.dim('does not exist!')}`);
+				throw new Error(`Template ${color.reset(tmpl)} ${color.dim('does not exist!')}`);
```

Comment:
This used to log the error and close the spinner before `ctx.exit` could be called. This throws so the higher scope can catch and log appropriately.